README.md

```diff
@@ -1,10 +1,23 @@
 # Varkes
-*Varkes* means small boats in contrast to the big ships. We are building a framework/toolkit of components, which then mock the big ships (applications). The aim is to give a framework to application writers that can mock their application so that they can experiment on Kyma functions without thinking about the deployment of their applications. Once they have the required logic they want from Kyma, they can deploy their applications without any change on Kyma side.
-## App Connector Client
-App connector has a REST api that is used to get certificate from kyma cluster. You can run it with `npm start` or you can use the *Dockerfile* to create a docker image. Further instructions are in [app-connector-client/README.md](app-connector-client/README.md).
 
-## OpenAPI Mock
-Mocks REST calls given an OpenAPI specification. [openapi-mock/README.md](openapi-mock/README.md).
+## Overview
 
-## Odata Mock
-Mocks rest calls given an OData specification [odata-mock/README.md](odata-mock/README.md).
\ No newline at end of file
+Varkes is Greek for "small boats", easy to operate and maintain. The Varkes framework is lightweight and user-friendly, but powerful enough to mock large external applications, or "big ships", and experiment with Kyma functionality without thinking about their deployment. Use the framework to set up the connection between Kyma and the mock application to deploy the applications without any change on Kyma side.
+
+
+The Varkes framework provides the following npm components to make the application mock work:

```

Comment:
npm is the package manager the component  is rather a "node" "module" which is managed by "npm", so just use "node module".
These components will make any application mock work, there is not "the" mock